app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt

```diff
@@ -24,27 +26,44 @@ import timber.log.Timber
 
 class TabFragment(index: Int) : Fragment() {
     var tabIndex = index
+    var articles : List<Article>? = null
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        if(activity != null){               // Afterwards keep displaying already initialized articles
```

Comment:
I think our convention should be to not use in-line comments, would rather it was on top of the code block its trying to explain